unfetter-discover/unfetter-ui
# assessments beta full results tab

supports unfetter-discover/unfetter#1177
supports unfetter-discover/unfetter#1068

## problem
assessment beta (v3) needs to show assessed phases for capability baseline assessments
assessment v2 edit wizard did not load

## changes
* correctly load the v2 edit wizard, by not calling the effects 2x
* pass a `isCapability` to the backend `risk-by-attackpattern` endpoint to populate the assessed phases
* Improved unassessed list repsonse time
  * added `trackBy` for all the lists
  * instead of querying the backend for unassessed attack patterns, I used the ngrx `config` store

## test
pull this pr
_Edit test_
1. edit a v2 assessment, the page should load

__ Test unassessed list response time __
1. load results tab, click around the tactis and see the unassessed attack patterns list load faster
_Load assessed test_
1. create a beta assessment using a capability
2. load the capability in the results tab
3. verify there are assessed capabilities shown
see screen shot

![screen shot 2018-06-20 at 2 02 12 pm](https://user-images.githubusercontent.com/30807406/41676108-1e576f58-74b4-11e8-936c-e1b65c514745.png)
